features/gebruik_in_lopende_tekst.feature

```diff
@@ -34,138 +34,126 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
   - Aanduiding naamgebruik "E" (eigen naam) wordt gehanteerd voor een persoon die geen actuele en geen ontbonden huwelijken/partnerschappen heeft gehad
 
   Abstract Scenario: gebruikInLopendeTekst bij geslacht <voorbeeld>
-    Gegeven het systeem heeft een persoon met de volgende gegevens
-    | naam                        | waarde     |
-    | burgerservicenummer         | 999992934  |
-    | geslachtsaanduiding (04.10) | <geslacht> |
-    En de persoon heeft de volgende 'naam' gegevens
-    | naam                                 | waarde  |
-    | voornamen (02.10)                    | Jo Rene |
-    | adellijke titel of predicaat (02.20) |         |
-    | voorvoegsel (02.30)                  |         |
-    | geslachtsnaam (02.40)                | Groenen |
-    | aanduiding naamgebruik (61.10)       | E       |
-    Als personen wordt gezocht met de volgende parameters
-    | naam                | waarde                            |
-    | type                | RaadpleegMetBurgerservicenummer   |
-    | burgerservicenummer | 999992934                         |
-    | fields              | adressering.gebruikInLopendeTekst |
-    Dan heeft de response een persoon met de volgende 'adressering' gegevens
-    | naam                  | waarde                  |
-    | gebruikInLopendeTekst | <gebruikInLopendeTekst> |
-
-    Voorbeelden:
-    | voorbeeld | geslacht | gebruikInLopendeTekst |
-    | man       | M        | de heer Groenen       |
-    | vrouw     | V        | mevrouw Groenen       |
-    | onbekend  | O        | J.R. Groenen          |
+    Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende gegevens
+      | naam                                 | waarde     |
+      | geslachtsaanduiding (04.10)          | <geslacht> |
+      | voornamen (02.10)                    | Jo Rene    |
+      | adellijke titel of predicaat (02.20) |            |
```

Comment:
In principe kan je deze en de volgende regel weglaten. Doen we volgens mij op alle andere plaatsen ook behalve daar waar het veld een lege waarde krijgt vanuit een voorbeeld tabel.